src/test/ReactDefaultPerf.js

```diff
@@ -121,6 +141,29 @@ var ReactDefaultPerf = {
       ReactDefaultPerfAnalysis.getTotalTime(measurements).toFixed(2) + ' ms'
     );
   },
+  /**
+   * prints table of components and their lifecycle counts
+   * @param  {array} measurements array of measurements taken by DefaultPerf
+   */
+  printLifecycles: function(measurements) {
+    measurements = measurements || ReactDefaultPerf._allMeasurements[
+      ReactDefaultPerf._allMeasurements.length - 1
+    ];
+    var components = measurements.components;
+    var summary = ReactDefaultPerfAnalysis.getLifecycleSummary(components);
+    console.table(summary.map(function(item) {
+      var result = {};
+      var keys = Object.keys(item);
+      keys.forEach(function(key) {
```

Comment:
thats a fair point. if whoever reviews the PR wants me to update it to use reduce i'm happy to do so. not clear if they would like to keep consistent with the other print methods already in the file.